File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.py
def solution(myStr):
    myStr = myStr.replace("a", " ").replace("b", " ").replace("c", " ").strip()
    answer = []
    myStr = myStr.split()
    for s in myStr:
        if s != " ":
            answer.append(s)
            
    return answer if answer else ["EMPTY"]
        

File: 프로그래머스/0/181862. 세 개의 구분자/세 개의 구분자.java
import java.util.*;

class Solution {
    public String[] solution(String myStr) {
        String[] strArr = myStr.replace("a", " ").replace("b", " ").replace("c", " ").split(" ");
        String[] arr = Arrays.stream(strArr).filter(s -> !s.isEmpty()).toArray(String[]::new);
        if(arr.length == 0) {
            return new String[] { "EMPTY" };
        }
        return arr;
    }
}